app-backup/luckybackup/luckybackup-0.4.9.recipe

```diff
@@ -0,0 +1,94 @@
+SUMMARY="...a powerful, fast and reliable backup & sync tool"
+DESCRIPTION="an application that backs-up and/or synchronizes any directories with the power of rsync.\
+It is simple to use, fast (transfers over only changes made and not all data),\
+safe (keeps your data safe by checking all declared directories before proceeding in any data manipulation ),\
+reliable and fully customizable."
+COPYRIGHT="2008-2017	Loukas Avgeriou	"
+LICENSE="GNU GPL v3"
+REVISION="1"
+HOMEPAGE="http://luckybackup.sourceforge.net"
+SOURCE_URI="https://sourceforge.net/projects/luckybackup/files/0.4.9/source/luckybackup-0.4.9.tar.gz"
+CHECKSUM_SHA256="c161e98a11de5ad0a32a8e060e3fd45695a93ea3adf0581764a6b5d32e8da23b"
+SOURCE_DIR="luckybackup"
+SOURCE_FILENAME="luckybackup-$portVersion.tar.gz"
+SOURCE_DIR="luckybackup-$portVersion"
+ADDITIONAL_FILES="luckybackup.rdef.in"
+
+ARCHITECTURES="x86_gcc2 x86_64"
+
+PROVIDES="
+	luckybackup$secondaryArchSuffix = $portVersion
+	app:LuckyBackup = $portVersion
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
```

Comment:
In strict mode haikuporter gives a policy error for this one
```diff
+	lib:libGL$secondaryArchSuffix
```

Reply:
@Begasus ..hm.. what should i do? maybe try to disableit and try rerun receipt?

Reply:
The package is requiring it, haven't checked if it actually uses GL, try disabling the devel package and rerun the recipe